src/CommunityToolkit.Maui.Core/Views/Popup/PopupOverlay.windows.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Displays Overlay in the Popup background.
+/// </summary>
+public class PopupOverlay : WindowOverlay
+{
+	IWindowOverlayElement popupOverlayElement;
+
+	/// <summary>
+	/// Instantiates a new instance of <see cref="PopupOverlay"/>.
+	/// </summary>
+	/// <param name="window">An instance of <see cref="IWindow"/>.</param>
+	/// <param name="overlayColor">Popup overlay color</param>
+	public PopupOverlay(IWindow window, Color? overlayColor = null) : base(window)
+	{
+		popupOverlayElement = new PopupOverlayElement(this, overlayColor);
+		AddWindowElement(popupOverlayElement);
+		EnableDrawableTouchHandling = true;
+	}
+
+	class PopupOverlayElement : IWindowOverlayElement
+	{
+		readonly IWindowOverlay overlay;
+		RectF overlayRect = new RectF();
+		Color overlayColor = Color.FromRgba(255, 255, 255, 153);
```

Comment:
probably add comment it is 60% background

Reply:
```suggestion
		Color overlayColor = Color.FromRgba(255, 255, 255, 153); // 60% Opacity 
```